cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -213,7 +215,7 @@ func Run(c *cloudcontrollerconfig.CompletedConfig, stopCh <-chan struct{}) error
 }
 
 // startControllers starts the cloud specific controller loops.
-func startControllers(c *cloudcontrollerconfig.CompletedConfig, stop <-chan struct{}, cloud cloudprovider.Interface) error {
+func startControllers(c *cloudcontrollerconfig.CompletedConfig, stop <-chan struct{}, cloud cloudprovider.Interface, controllers map[string]InitFunc) error {
```

Comment:
nit: this change belong into the other commit.

Reply:
fixed